src/lua_hooklib.c

```diff
@@ -768,4 +770,27 @@ boolean LUAh_HurtMsg(player_t *player, mobj_t *inflictor, mobj_t *source)
 	return hooked;
 }
 
+void LUAh_NetArchiveHook(lua_CFunction archFunc)
```

Comment:
Got confused in stack math, passed wrong thing to Lua. X_X;;;

It's supposed to pass archFunc with an upvalue of tables, not tables itself!